backend/backend.go

```diff
@@ -50,26 +46,20 @@ func (id ID) String() string {
 	}
 }
 
-// ProverOption defines option for altering the behaviour of the prover in
+// ProverOption defines option for altering the behavior of the prover in
 // Prove, ReadAndProve and IsSolved methods. See the descriptions of functions
 // returning instances of this type for implemented options.
 type ProverOption func(*ProverConfig) error
 
 // ProverConfig is the configuration for the prover with the options applied.
 type ProverConfig struct {
-	Force         bool                      // defaults to false
```

Comment:
some nonsense :) since in provers we first solve the constraint system and then compute the proof elements, at some point we wanted to benchmark provers without bothering to have a valid witness, and, test the verifier with invalid proof. Essentially saying to the verifier "here is some garbage, please reject it"